chart/templates/iq-server-deployment.yaml

```diff
@@ -221,7 +221,7 @@ spec:
         {{- end }}
       initContainers:
         - name: {{ .Release.Name }}-set-iq-persistence-ownership
-          image: busybox:1.28
+          image: {{ include "nexus-iq-server-ha.busyboxImage" . }}
```

Comment:
Thanks for this suggestion.

There's a couple of other places where we use busybox which I think we'll also need to change:

https://github.com/sonatype/nexus-iq-server-ha/blob/53cfbb3daddd82bb5a24dd82c178b0c052da0797/chart/templates/delete-old-aggregate-logs-cronjob.yaml#L21

https://github.com/sonatype/nexus-iq-server-ha/blob/450c35fc9eb182d37a69dffa1dd204e68f3dea6b/chart/values.yaml#L630